### DISCUSSION TITLE
XBRL and RDF Support ?

### DISCUSSION THREAD

**Discussion by JJDSNT** [Unstructured-IO/unstructured#751] [Ideas]
Hi, it would make sense to have a xbrl and RDF support?

https://gitlab.xbrl.org/xii/ixbrl-samples

https://rdflib.dev/

Regards,

**Comment by MthwRobinson** [comment]
@JJDSNT - Thanks for the suggestions! Just added #839 to cover RDF files. For XBRL, you may find our [SEC filings repo](https://github.com/Unstructured-IO/pipeline-sec-filings/tree/main) helpful. You may also be able to process those with `partition_xml`.
